CHANGELOG.md

```diff
@@ -2,7 +2,12 @@
 
 ### Note
 
-- A new flag `disableBackgroundScan` to disable kyverno background scans (default value is `false`). When this is enabled, kyverno will not perform background scans and won't trigger continuous evaluation of policies.
+- A new flag `backgroundScan` to enable/disable kyverno background scans (default value is `true`). When this is set to `false`, kyverno will not perform background scans and won't trigger continuous evaluation of policies.
+- A new flag `admissionReports` to enable/disable kyverno admission reports (default value is `true`). When this is set to `false`, kyverno will not create admission reports.
+- If both `backgroundScan` and  `admissionReports` are set to `false` the entire reports system will be disabled.
+- A new flag `reportsChunkSize` to split reports according to the number of results contained in the report (default value is `1000`). This can be disabled by setting the flag value to `0`.
+- The `splitPolicyReport` was removed, splitting reports per policy is always enabled.
```

Comment:
Would this break the backward compatibility? What if users have this flag set?

Reply:
Added back, fixed changelog and added a message at kyverno startup.